src/main/webapp/add_event_page/add_event.html

```diff
@@ -4,43 +4,51 @@
     <meta charset="UTF-8">
     <title>Add Event</title>
     <link rel="stylesheet" href="style.css">
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs&libraries=places"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="initMap()">
+  <body>
     <div id="header">
       <h1>Add New Event</h1>
     </div>
-
-    <div id="form">
-        <form action="/new_event.php">
+        <form id="form" method="POST">
```

Comment:
This looks great, @mingyi09! There's just one last thing, with the method of redirection I suggested earlier, the tab always asks to "Confirm form resubmission" when refreshing the directory page after submitting an event so I found another way that doesn't cause this. Sorry about changing what I said earlier. I'll put comments where it needs to be changed.

The form `method="POST"` here would be removed.